### PR TITLE
Spec writing - Brian Provost

### PR DESCRIPTION
**Pull request by provostprovost**
Exercises, RPS, and Extension.

### CODE REVIEW

File: exercises.rb [hunk 1]
```diff
@@ -107,6 +164,37 @@ def start
...
+    @player1 = gets.chomp
+    puts "Input name of Player 2:"
+    @player2 = gets.chomp
+
+    #Create new RPS class with player names
+    @rps = RPS.new(player1,player2)
+
+    # Default play_again for while loop
+    play_again = 'y'
+
```
Comment (prudepixie) [inline exercises.rb:181]: Interesting, I didn't think you can do it this way

File: exercises.rb [hunk 2]
```diff
@@ -107,6 +164,37 @@ def start
...
+      # Play the game
+      puts @rps.play(@player1_move, @player2_move)
+
+      #Ask to play again
+      puts "Play again? (y/n)"
+      play_again = gets.chomp.downcase
+    end
   end
 end
```
Comment (prudepixie) [inline exercises.rb:200]: Your code is really easy to follow and very organized, 
especially with the comments on the side

File: exercises.rb [hunk 3]
```diff
@@ -66,6 +79,7 @@ def self.ex8(people)
   # Hint: Google for the wikipedia article on leap years
   def self.ex9(time)
     # TODO
+    time.year % 4 == 0 ? true : false
   end
```
Comment (prudepixie) [inline exercises.rb:83]: (I know it's irrelevant to RPS) I like this piece of code here, much easier done than mine

File: exercises.rb [hunk 4]
```diff
@@ -83,11 +97,55 @@ class RPS
...
+
+  def initialize(player1, player2)
+    @player1 = player1
+    @player2 = player2
+    @player1_wins = 0
+    @player2_wins = 0
+    # @called_count only exists for testing purposes
+    @called_count = 0
+  end
+
```
Comment (prudepixie) [inline exercises.rb:113]: I'm interested to know how you made player1_move and player2_move without having put it under attr_accessor

File: exercises.rb (hunk 3 above)
Comment (gilbert) [inline exercises.rb:83]: [Actually it's not totally correct](http://en.wikipedia.org/wiki/Leap_year#Algorithm), but it's sufficient for the next 85 years :)

File: exercises.rb (hunk 4 above)
Comment (gilbert) [inline exercises.rb:113]: The next line (`def play(player1_move, player2_move)` accepts the moves when the `play` method is called. The player moves are only relevant for that one play, so there's no need to store them via an `attr_accessor`

**Comment by gilbert** [comment]
Great job, Brian

**Pull request closed without merging**